visualizer/index.html

```diff
@@ -128,37 +129,37 @@
       position: relative;
       text-align: center;
     }
-    .pexpr > .label .caseName::before {
+    .pexpr > .self > .label .caseName::before {
       content: '\2014';
       padding: 0 0.5em;
     }
-    .pexpr > .label .caseName {
+    .pexpr > .self > .label .caseName {
       font-style: italic;
       opacity: 0.7;
     }
-    .pexpr:not(.failed) > .label:not(.prim):hover {
+    .pexpr:not(.failed) > .self:not(.nextStep) > .label:not(.prim):hover {
```

Comment:
Is there a way you can do this in the semantics editor CSS instead of here? Since it's specific to the semantics editor, it doesn't really belong here.